packages/api-explorer/src/Doc.jsx

```diff
@@ -263,10 +263,11 @@ class Doc extends React.Component {
 
     return (
       operation &&
-      operation.responses && (
+      operation.schema &&
+      operation.schema.responses && (
```

Comment:
I'd add a `hasResponses` method into `oas`, but since responses are required to be present to be a valid OAS definition I don't think it makes sense to add to account for legacy specs in our DB that might not be valid.